"Solution"

isikach opened the pull request.

isikach committed:
"Solution"

isikach committed:
"Solution"

isikach committed:
Merge branch 'develop' of https://github.com/isikach/py-taxi-service-search-and-tests into develop

isikach committed:
Merge branch 'master' into develop

isikach committed:
"Solution"

isikach committed:
Merge remote-tracking branch 'origin/develop' into develop

isikach committed:
"Solution"

isikach committed:
"Solution"

Y-Havryliv commented on taxi/tests/test_search.py:
```diff
@@ -0,0 +1,114 @@
+        )
+        self.client.force_login(self.user)
+
+        Driver.objects.create(
```
think about creating these instances more efficient

Y-Havryliv commented on taxi/tests/test_view.py:
```diff
@@ -0,0 +1,43 @@
+        response = self.client.get(reverse("taxi:driver-list"))
+        self.assertEquals(response.status_code, 200)
+        drivers = Driver.objects.all()
+        self.assertEquals(len(drivers), len(response.context["driver_list"]))
```
you could use `assertQuerySetEqual` for this purpose. And this will test if the results actually match

isikach committed:
add blank line in requirements.txt
creating instances in test using cycle for
rewrite test using assertQuerySetEqual
